storage/store_test.go

```diff
@@ -0,0 +1,213 @@
+package storage
+
+import (
+	"database/sql"
+	"fmt"
+	"testing"
+
+	"github.com/pavolloffay/jaeger-clickhouse/storage/clickhousedependencystore"
+	"github.com/pavolloffay/jaeger-clickhouse/storage/clickhousespanstore"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/hashicorp/go-hclog"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/pavolloffay/jaeger-clickhouse/storage/clickhousespanstore/mocks"
+)
+
+const (
+	testIndexTable        = "test_index_table"
+	testSpansTable        = "test_spans_table"
+	testOperationsTable   = "test_operation_table"
+	testSpansArchiveTable = "test_spans_archive_table"
+)
+
+var errorMock = fmt.Errorf("error mock")
+
+func TestStore_SpanWriter(t *testing.T) {
+	db, _, err := getDbMock()
+	require.NoError(t, err, "an error was not expected when opening a stub database connection")
```

Comment:
nit: the error descriptions are useful but in the simple cases they do not bring much value